sdk/appcenter/src/main/java/com/microsoft/appcenter/http/HttpException.java

```diff
@@ -20,6 +22,11 @@
      */
     private final String payload;
 
+    /**
+     * HTTP headers.
+     */
+    private Map<String, String> headers = new HashMap<>();
```

Comment:
don't init variable here since it can be set via constuctor.
Remove the = and put final.

Reply:
I don't see the fix after comment resolved.